package.json

```diff
@@ -44,23 +44,23 @@
     "husky": "7.0.4",
     "jest": "27.5.1",
     "license-checker": "25.0.1",
-    "lint-staged": "12.4.1",
+    "lint-staged": "12.5.0",
     "log-symbols": "4.1.0",
     "node-fetch": "2.6.7",
-    "npm-check-updates": "12.2.0",
+    "npm-check-updates": "12.5.12",
     "npm-run-all": "4.1.5",
-    "prettier": "2.6.2",
+    "prettier": "2.7.1",
     "rimraf": "3.0.2",
-    "semver": "7.3.7",
+    "semver": "7.3.8",
     "standard-version": "9.5.0",
     "terminal-link": "2.1.1",
     "ts-config-webpack-plugin": "2.0.3",
```

Comment:
this config plugin is webpack 4 only

we should configure webpack 5 without this plugin